packages/sdk/src/transactions/Transaction/finalized.ts

```diff
@@ -0,0 +1,157 @@
+ */
+export function finalize(transaction: Signable): Finalized {
+    switch (transaction.version) {
+        case 0:
+            return AccountTransactionV0.create(preFinalized(transaction), transaction.signature);
+        case 1:
+            return AccountTransactionV1.create(preFinalized(transaction), transaction.signatures);
+    }
+}
+
+/**
+ * Gets the transaction hash that is used to look up the status of a transaction.
+ * @param transaction the transaction to hash
+ * @returns the sha256 hash of the serialized block item kind, signatures, header, type and payload
+ */
+export function getAccountTransactionHash(transaction: Finalized): Uint8Array {
+    switch (transaction.version) {
+        case 0:
+            return AccountTransactionV0.getAccountTransactionHash(transaction);
+        case 1:
+            return AccountTransactionV1.getAccountTransactionHash(transaction);
+    }
+}
+
+/**
+ * Serializes a _finalized_ transaction as a block item for submission to the chain.
+ * @param transaction the signed transaction to serialize
+ * @returns the serialized block item as a byte array
+ */
+export function serializeBlockItem(transaction: Finalized): Uint8Array {
+    switch (transaction.version) {
+        case 0:
+            return AccountTransactionV0.serializeBlockItem(transaction);
+        case 1:
+            return AccountTransactionV1.serializeBlockItem(transaction);
+    }
+}
+
+/**
+ * Deserializes a transaction from the block item encoding used to send it to a node.
+ *
+ * @param buffer A buffer containing transaction encoding. It is expected to start the _block item kind_.
+ * @returns a block item.
+ **/
+export function deserializeBlockItem(buffer: ArrayBuffer): Upward<BlockItem> {
+    const cursor = Cursor.fromBuffer(buffer);
+
+    const blockItemKind = deserializeUint8(cursor);
+    let blockItem: BlockItem;
+    switch (blockItemKind) {
+        case BlockItemKind.AccountTransactionKind:
+            blockItem = {
+                kind: BlockItemKind.AccountTransactionKind,
+                transaction: AccountTransactionV0.deserialize(cursor),
+            };
+            break;
+        case BlockItemKind.CredentialDeploymentKind:
+            blockItem = {
+                kind: BlockItemKind.CredentialDeploymentKind,
+                transaction: deserializeCredentialDeployment(cursor),
+            };
+            break;
+        case BlockItemKind.UpdateInstructionKind:
+            throw new Error('deserialization of UpdateInstructions is not supported');
+        case BlockItemKind.AccountTransactionV1Kind:
+            blockItem = {
+                kind: BlockItemKind.AccountTransactionV1Kind,
+                transaction: AccountTransactionV1.deserialize(cursor),
+            };
+            break;
+        default:
+            return null;
+    }
+
+    if (cursor.remainingBytes.length !== 0) throw new Error('Deserializing the transaction did not exhaust the buffer');
+    return blockItem;
+}
```

Comment:
Maybe this should be added into the implementation done in https://github.com/Concordium/concordium-node-sdk-js/pull/615 instead..